docs/_peoples/hailiang-wang.md

```diff
@@ -1,12 +1,21 @@
 ---
-name: Hain WANG (王海良)
+name: Hai Liang WANG (王海良)
 site: https://github.com/Samurais
-avatar: /assets/peoples/hailiang-wang/avatar.jpg
-bio: ChatOpera 联合创始人 & CEO, 微软人工智能最有价值专家
-email: 
+avatar: /assets/peoples/hailiang-wang/avatar.png
```

Comment:
I saw you had changed your avatar image file from a 20KB jpg to 400KB png, which will slow your profile page load.

Do you make sure you want this?

My suggestion would be using a JPG 500x500 image because it will have enough quality, and we only show the image file for a 160x160 size maximum on our page.

Reply:
Images are compressed with JPG format in the latest commit.